refinery/ui/source/js/commons/services/openid-token.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
common service should have a unit test, pretty cookie cutter.
Also not wrapped in iffy or use the other service stylings https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y060